torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -750,14 +750,12 @@ def pad_bounding_box(
 
     left, right, top, bottom = _parse_pad_padding(padding)
 
-    bounding_box = bounding_box.clone()
-
     # this works without conversion since padding only affects xy coordinates
-    bounding_box[..., 0] += left
-    bounding_box[..., 1] += top
     if format == features.BoundingBoxFormat.XYXY:
-        bounding_box[..., 2] += left
-        bounding_box[..., 3] += top
+        pad = torch.tensor([left, top, left, top], device=bounding_box.device)
+    else:
+        pad = torch.tensor([left, top, 0, 0], device=bounding_box.device)
+    bounding_box = bounding_box + pad
```

Comment:
Not blocking: instead of repeating the tensor creation, we could only set the values there:

```suggestion
        pad = [left, top, left, top]
    else:
        pad = [left, top, 0, 0]
    bounding_box = bounding_box + torch.tensor(pad, device=bounding_box.device)
```